### DISCUSSION TITLE
Make it so spinners display the max number of points that are gainable from the spinner when it first appears

### DISCUSSION THREAD

**Discussion by IVII1** [ppy/osu#25172] [Ideas] [answered]
With the new spinner changes, I always keep spinning out of habit even though the spinner is maxed out, I think players would benefit if you showed the maximum amount of points attainable through a spinner so people can stop spinning when they reach it and stay sharp for upcoming objects.

**Comment by peppy** [comment]
I wanted to show a "MAX" text after they are maxed. Would this work for you? I don't really think showing the max score will look good.

**Reply by peppy** [reply]
I'm going to take a look at making this change.

**Reply by IVII1** [reply]
Yeah that works. Basically just something to tell me i can stop spinning. Maybe even add sfx

**Reply by peppy** [reply, marked as answer]
#25177
